en/docs/modules/misc/MagneticConnector.md

```diff
@@ -2,17 +2,36 @@
 
 <img src="https://docs.retrogadgets.game/api/modules/MagneticConnector.png" width="200" align="right">
 
-Magnetic connects allow separated boards to connect and disconnect at set points. Unfortunately, magnetic connectors are essentially just magnets and do not report what they're connected to. Boards connected to each other with a powered magnet (light green, toggleable with the button) will be picked up as one.
-
-ℹ️ By knowing the state of all connectors, it is possible in certain configurations to know the arrangement of connectable boards.
+Magnetic connects allow separated boards to connect and disconnect at set points. Boards connected to each other with a powered magnet (light green, toggleable with the button) will be picked up as one.
 
 ## Properties
 
+### AttachedConnector - `MagneticConnector` **[Read only]**
+The other `MagneticConnector` connected to this one.
+
 ### ButtonState - `boolean` **[Read only]**
 The state of the physical button on the connector. `true` if the button is being held.
 
 ### IsConnected - `boolean` **[Read only]**
-`true` if the connector is connected to another one. The only indication of connectivity. 
+`true` if the connector is connected to another one. The only indication of connectivity.
+
+### Motherboard - `Motherboard` **[Read only]**
+The `Motherboard` attached with this `MagneticConnector`.
+
+### Type - `string` **[Read only]**
+Always return `MagneticConnectorEvent`.
+
+## Event - `MagneticConnectorEvent`
+### For Retro Gadgets V0.1.5+
+The event emitted as part of the [CPU](./CPU.md) event system.
+
+Sent when a `MagneticConnector` is connected to another.
+
+### IsConnected - `boolean` **[Read only]**
+`true` if the connector is connected to another one. The only indication of connectivity.
+
+### Type - `string` **[Read only]**
+Always return `MagneticConnectorEvent`.
```

Comment:
Oh ok, I though this was a new thing.

Reply:
Do I remove in the `Properties` section too ?

Reply:
> Do I remove in the `Properties` section too ?

I actually have no idea! That one _is_ new. As all modules have board (I think?) and type now, it might be best to make a general document about the things every module has.

Reply:
I've opened #53.

Reply:
Ok, I think it's better without the `Type` and `Motherboard` everywhere.

Reply:
Perhaps? You can discuss this further in #53, and we can get opinions from the rest of the community.

Reply:
I have severely misread your message to read "with" instead of "without".